packages/backend/src/managers/playgroundV2Manager.ts

```diff
@@ -312,6 +315,11 @@ export class PlaygroundV2Manager implements Disposable {
     return name;
   }
 
+  private isFreeName(name: string): boolean {
+    const names = new Set(this.getConversations().map(c => c.name));
+    return !names.has(name);
+  }
```

Comment:
```suggestion
  private isFreeName(name: string): boolean {
    return !this.getConversations().some(c => c.name === name)
  }
```

I do not think you need to use a `Set` here